interfaces/cython/cantera/func1.pyx

```diff
@@ -87,3 +100,78 @@ cdef class Func1:
 
     def __copy__(self):
         raise NotImplementedError('Func1 object is not copyable')
+
+
+cdef class Tabulated1(Func1):
```

Comment:
Can we come up with another, more descriptive, name for this than `Tabulated1`? I assume the base class is called `Func1` because it takes one argument, but I don't think we necessarily need to carry that convention into derived classes. In most cases, it is not necessary to explicitly call `Func1`, so I don't think most users will see that name. However, this class must be called by a user to set it up, so a better name would help with discoverability.

Reply:
I changed this to `TabulatedFunction`.